_episodes/09-cmorization.md

```diff
@@ -0,0 +1,1053 @@
+>
+>
+> Concretely, the CMOR standards dictate e.g. the variable names and units,
+coordinate information, how the data should be structured (e.g. 1 variable per
+file), additional metadata requirements, but also file naming conventions a.k.a.
+the data reference syntax (DRS). All this information is stored in so-called
+CMOR tables. As example, the CMOR tables for the CMIP6 project can be found
+[here](https://github.com/PCMDI/cmip6-cmor-tables).
+{: .callout}
+
+The Earth System Grid Federation (ESGF) is home to all CMIP data. The data
+hosted there typically (mostly) follow the CMIP standards, and therefore
+ESMValTool should work with these data without problems.
+
+Datasets that are *not* part of one of the CMIP projects often don't follow the
+CMOR standards. In this case, a reformatting script can be used to create a
+CMOR-compliant copy of these datasets. CMORizer scripts for several popular
+datasets are included in ESMValTool, and ESMValTool also provides a convenient
+way to execute them.
+
+Occasionally it happens that there are still minor issue with CMIP datasets. In
+those cases, it is possible to fix those issues in ESMValCore before any further
+processing is done. The same can be done for non-CMIP data. The advantage is
+that you don't need to store an additional, reformatted copy of the data. The
+disadvantage is that these fixes should be implemented inside ESMValCore.
+Writing a CMORizer script is technically is simpler.
+
+The concepts discussed so far are illustrated in the figure below.
+![Data flow with ESMValTool](../fig/data_flow.png)
+*Illustration of the data flow in ESMValTool.*
+
+In this lesson, we will re-implement a CMORizer script for the FLUXCOM dataset
+that contains observations of the Gross Primary Production (GPP), a variable
+that is important for calculating components of the global carbon cycle. We will
+go through all the steps and explain relevant topics as we go. If you prefer to
+implement CMOR fixes, please read the documentation
+[here](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/develop/fixing_data.html#fixing-data).
+While fixes are implemented slightly differently, conceptually the process is
+the same and the concepts explained in this episode are still useful.
+
+## Obtaining the data
+
+> ## Get the data
+> The data for this episode is available via the [FluxCom Data
+> Portal](http://www.bgc-jena.mpg.de/geodb/BGI/Home). First you'll need to
+> register. After registration, in the dropdown boxes, select FLUXCOM as the data
+> choice and click download. Three files will be displayed. Click the download
+> button on the "FLUXCOM (RS+METEO) Global Land Carbon Fluxes using CRUNCEP
+> climate data". You'll be send an FTP address to access the server. Connect
+> to the server, follow the path in your email, and look for the file
+> `raw/monthly/GPP.ANN.CRUNCEPv6.monthly.2000.nc`. Download that file and save in
+> in a folder called `/RAWOBS/Tier3/`.
```

Comment:
The `/RAWOBS/Tier3/` is a directory rather than a folder name here. Can we make it clear by re-phrasing it?

Reply:
Not sure I understand. What's the difference between a directory and a folder name?

Reply:
Because the folder name is `FLUXCOM` here, I thought the sentence “save it in a folder called `/RAWOBS/Tier3/FLUXCOM` " might be confusing from a user’s  (researcher) perspective as `/RAWOBS/Tier3/FLUXCOM ` shows three folders. However, from a developer’s perspective, I am aware that a folder is identified in this way. I see several cases in the lesson that a file or a folder is called in this way. So, it is fine with me.